fix: markdown editor auto scrolls

nguyengiang2000 opened the pull request:
What does it do?
This pull request fixes an issue where the Markdown Editor auto-scrolls to the top when applying formatting (e.g., bullet lists, headings). The editor's position is maintained during formatting operations, improving the editing experience.

Why is it needed?
The auto-scrolling issue disrupts the user's workflow by jumping to the top of the page when formatting is applied. This fix ensures that the editor maintains its position, allowing users to continue editing without unnecessary scrolling.

How to test it?
1.Open the Markdown Editor in the application.
2.Create or open a document with some content.
3.Apply different types of formatting (e.g., bullet lists, headings) to the content.
4.Verify that the editor does not scroll to the top during or after applying the formatting.

Related issue(s)/PR(s)
Fixes https://github.com/strapi/strapi/issues/20160.

Let us know if this is related to any issue/pull request

hanpaine commented:
Hey @nguyengiang2000 thanks very much for your PR! We will review it soon 🙏

nguyengiang2000 commented:
Thank you for your response.

On Tue, Oct 1, 2024 at 8:10 AM Hannah Paine ***@***.***>
wrote:

> Hey @nguyengiang2000 <https://github.com/nguyengiang2000> thanks very
> much for your PR! We will review it soon 🙏
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/strapi/strapi/pull/21434#issuecomment-2386274265>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQDZJLZ2TKZRHDCZFWCFJOTZZK3N7AVCNFSM6AAAAABOXDZGAOVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDGOBWGI3TIMRWGU>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

derrickmehaffy commented:
Hi @nguyengiang2000 it looks like there is a lot of conflicts here and I suspect this fix was originally based on a Strapi 4 branch rather than Strapi 5. Can you rebase your changes on the current development branch.

Thanks,

nguyengiang2000 commented:
Can you tell me more specificly what should i do ?

On Mon, Oct 7, 2024 at 11:36 AM DMehaffy ***@***.***> wrote:

> Hi @nguyengiang2000 <https://github.com/nguyengiang2000> it looks like
> there is a lot of conflicts here and I suspect this fix was originally
> based on a Strapi 4 branch rather than Strapi 5. Can you rebase your
> changes on the current development branch.
>
> Thanks,
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/strapi/strapi/pull/21434#issuecomment-2397622017>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQDZJLYYA7HE7FC237IKNZDZ2LID3AVCNFSM6AAAAABOXDZGAOVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDGOJXGYZDEMBRG4>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

Marc-Roig commented:
@nguyengiang2000 we modified the base branch (we moved the v4 code into "v4")
we will be reviewing this pr during this week/next one

nguyengiang2000 commented:
Awesome, thanks for the update!

On Thu, Oct 24, 2024 at 6:11 AM Marc Roig ***@***.***> wrote:

> @nguyengiang2000 <https://github.com/nguyengiang2000> we modified the
> base branch (we moved the v4 code into "v4")
> we will be reviewing this pr during this week/next one
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/strapi/strapi/pull/21434#issuecomment-2435259977>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQDZJL4VTV5VATP5EISGPMTZ5DWZDAVCNFSM6AAAAABOXDZGAOVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDIMZVGI2TSOJXG4>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

remidej commented on .github/actions/check-pr-status/dist/index.js:
I assume this diff comes from us juggling with different branches on this PR — could you revert it? There shouldn't be any change here

remidej reviewed (commented):
Hello, sorry for the delay but I got finally got around to testing your fix, and it works nicely 👌 

There's just 1 change that breaks the CI, I can merge once you revert it
